### ISSUE TITLE
Removing some route data in the Strip Section (Minor)

### ISSUE THREAD

**Issue by danlovett** [GoodCrossing/StripMaker#6]
Hello!

### Regards to the Route section
In the routes section of the strip, it includes all route data. Sometimes when a reroute is given, '[Airport ICAO]/[Departure Runway]'. In the real world, I doubt they would include this.

### The Suggestion
To find (if it is included) the part mentioned above and remove it. Maybe a search for if it's there in the first place?

### Screenshots
The Section related: 
![EuroScope_hMB5x7MTnd](https://user-images.githubusercontent.com/51034848/103468434-da4af500-4d50-11eb-96d9-a5bb084fdd9c.png)

:)

**Comment by GoodCrossing** [comment]
Hi! This is planned to be released alongside printer support. From the info that I have, the part of the route that is included is the first fix and the first airway. This is a bit hard to do properly as I have to get ES to parse the route before making the text, which is why it's like this temporarily. 

By the way, the current behaviour for long text in a field (a long callsign or route, for example) is to just keep writing to the right. This was chosen as it's the easiest to implement (🤪) and as it seems like it's the behaviour found IRL . However, let me know if you'd prefer it a different way (cut off, etc.)

**Comment by danlovett** [comment]
Okay. It seems neater if the route was to be on separate lines (in the route box). I will see if there is any screenshots to back this (doubt this isn't the case). I will update this comment accordingly :)

**Comment by GoodCrossing** [comment]
> Okay. It seems neater if the route was to be on separate lines (in the route box).

I think that's used in enroute strips, while the Fix + Airway is on departure strips. I think I'll implement the route on departure strips as I've said, and, since custom strips will be supported, I'll try to add the multiple line as an option.

**Comment by danlovett** [comment]
Oooo nice - Looking forward to it :D

**Comment by danlovett** [comment]
> I think that's used in enroute strips

Oooo just had an idea - not sure if it's on the agenda - making different strips for different points on the flight.
Having en route and arrival strips will also be cool :)

**Comment by GoodCrossing** [comment]
Yep - in the Features list you can see 'Customizable strip formats' is planned. This allows for Arrival/Enroute strips and I plan to have the plugin determine the best one for the current phase of flight (using the advanced print menu to override if needed). The way I've programmed the strips allows me to have customizable strips (for example if someone wants to adapt this plugin for FAA strips) and strips for other points in the flight.
